docs/components/tree-list.md

```diff
@@ -215,6 +215,26 @@ Boolean to hide icons. Defaults to `false`.
 <KTreeList hide-icons :items="items" />
 ```
 
+### collapsible
+
+Boolean to enable/disable collapse feature. Defaults to `false`.
+
+<KTreeList :collapsible="true" :items="collapsibleItems" />
+
+```html
+<KTreeList :collapsible="true" :items="items" />
+```
+
+### collapseAllInitially
```

Comment:
Trying to name consistent to other "initial" props we have, can we rename to `initial-collapse-all`